src/main/java/com/gepardec/training/microprofile/TrainingMetric.java

```diff
@@ -1,6 +1,7 @@
 package com.gepardec.training.microprofile;
 
 import org.eclipse.microprofile.metrics.annotation.*;
+import org.eclipse.microprofile.opentracing.Traced;
```

Comment:
Import not used, is this a dead import or are we missing something here?

Reply:
now code is organized and formatted